platform/mds/mds-web/src/main/resources/webapp/js/util.js

```diff
@@ -201,6 +201,86 @@
                        return scope.msg("mds.form.operator.none");
                    }
                };
+            },
+            
+            moveItems : function($scope, index) {
```

Comment:
I feel like these names are to generic for a class called MDSUtils. In a vacuum, what does moveItems in MdsUtils mean? I would rather expect this method to be either called moveItemsInTheSchemaList or moveItems in the MdsSchemaListUtils (I made up the schema list part, since I'm not sure what does this do).